### PR TITLE
Added Frontend developer skills 

### PR DESCRIPTION
**Pull request by vikas780**
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the contributing guidelines of the project.
- [x] The PR is made to the branch that's **called** "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] Have you followed the guidelines in the Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/CSwala/awesome-computer-science/pulls) for the same tech?
- [x] Is your submission is according to the desired pattern ?
- [x] Have you placed the contents in the `Readme.md` keeping the sorting order in mind ?
- [x] Have you placed the contents in the `Dictionary.json` keeping the sorting order in mind ?

Does you submission include -

- [x]	Category/Type
- [x]	Official Documnetation  
- [x]	Best YouTube creators  
- [x]	Useful Websites  
- [x]	Dedicated Github page 
- [x]	Best Courses  
- [x]	Ohter Tips / Hacks

### CODE REVIEW

File: Dictionary.json [hunk 1]
```diff
@@ -2338,7 +2338,76 @@
 		"Tips5" : "https://github.com/ibhavikmakwana/FlutterDartTips",
 		"Tips6" : "https://github.com/leisim/awesome-flutter-packages",
 		"Tips7" : "https://medium.com/@diegoveloper"
-	},	
+	},
+{
+    "Title": "Frontend Developer",
```
Comment (riturajjain2000) [inline Dictionary.json]: change this to "Frontend Development"

File: README.md [hunk 2]
```diff
@@ -2948,6 +2948,89 @@ https://github.com/leisim/awesome-flutter-packages<br>
 https://medium.com/@diegoveloper<br>
 <br>
 </details>
+<details>
+<summary>Frontend Developer</summary>
```
Comment (riturajjain2000) [inline README.md]: here also

File: README.md [hunk 3]
```diff
@@ -2948,6 +2948,89 @@ https://github.com/leisim/awesome-flutter-packages<br>
...
+<br>Category/Type - <br>Web development<br>
+<a href="https://raw.githubusercontent.com/kamranahmedse/developer-roadmap/master/img/frontend.png">
```
Comment (riturajjain2000) [inline README.md]: this image should be at last

File: Dictionary.json [hunk 4]
```diff
@@ -2338,7 +2338,76 @@
...
+    "Tag": "Web development",
+    "Documentation": "https://devdocs.io/html/",
```
Comment (riturajjain2000) [inline Dictionary.json:2345]: There should be only 1 documentation
rest should be under 'Website' tag

File: README.md [hunk 5]
```diff
@@ -2948,6 +2948,89 @@ https://github.com/leisim/awesome-flutter-packages<br>
...
+<img align="center" src="https://raw.githubusercontent.com/kamranahmedse/developer-roadmap/master/img/frontend.png"/>
+</a>
+
+<br>Official Documentation - <br>
+https://devdocs.io/html/<br>
```
Comment (riturajjain2000) [inline README.md:2956]: here also only single documentation

File: README.md [hunk 6]
```diff
@@ -2948,6 +2948,89 @@ https://github.com/leisim/awesome-flutter-packages<br>
...
+
+https://www.freecodecamp.org/news/what-is-tailwind-css-and-how-can-i-add-it-to-my-website-or-react-app/<br>
+https://designshack.net/articles/css/5-steps-to-drastically-improve-your-css-knowledge-in-24-hours/<br>
+https://dev.to/zandershirley/10-practical-javascript-tricks-2b7h<br>
+https://html.com/wp-content/uploads/html-cheat-sheet.pdf<br>
+https://digital.com/tools/html-cheatsheet/<br>
+https://www.sitepoint.com/top-15-jquery-tricks/<br>
+
+<br>
+<strong>Keep learning, Don't stop here only!  </strong>
```
Comment (riturajjain2000) [inline README.md]: this is not required

File: Dictionary.json [hunk 7]
```diff
@@ -2338,7 +2338,76 @@
 		"Tips5" : "https://github.com/ibhavikmakwana/FlutterDartTips",
 		"Tips6" : "https://github.com/leisim/awesome-flutter-packages",
 		"Tips7" : "https://medium.com/@diegoveloper"
-	},	
+	},
+{
+    "Title": "Frontend Developer",
+    "Tag": "Web development",
```
Comment (riturajjain2000) [inline Dictionary.json:2344]: change  category /type to "Development"  only as frontend is also there is other applications also

File: README.md [hunk 8]
```diff
@@ -2948,6 +2948,89 @@ https://github.com/leisim/awesome-flutter-packages<br>
 https://medium.com/@diegoveloper<br>
 <br>
 </details>
+<details>
+<summary>Frontend Developer</summary>
+<br>Category/Type - <br>Web development<br>
```
Comment (riturajjain2000) [inline README.md:2953]: change  category /type to "Development"  only as frontend is also there is other applications also

File: README.md [hunk 9]
```diff
@@ -2948,6 +2948,89 @@ https://github.com/leisim/awesome-flutter-packages<br>
...
+https://www.udemy.com/course/the-complete-javascript-course/<br>
+https://www.udemy.com/course/advanced-css-and-sass/<br>
+https://www.udemy.com/course/complete-guide-of-tailwind-css-/<br>
+https://www.udemy.com/course/bootstrap-4-from-scratch-with-5-projects/<br>
+https://www.udemy.com/course/jquery-course/<br>
+
+<br>Other Tips / Hacks - <br>
+https://dev.to/simonholdorf/10-tips-tricks-that-will-make-you-a-better-reactjs-dev-4fhn<br>
+https://www.telerik.com/blogs/10-time-saving-css-tips-i-learned-the-hard-way-when-using-sass<br>
+
```
Comment (riturajjain2000) [inline README.md]: remove this extra blank line

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]

**Comment by vikas780** [comment]
ok. I will change the file as per your need.

**Pull request closed without merging**